pkgs/top-level/all-packages.nix

```
@@ -25053,6 +25053,8 @@ with pkgs;
 
   cozy-drive = callPackage ../applications/networking/cozy-drive {};
 
+  cozy-drive = callPackage ../applications/networking/cozy-drive {};
+
```

We only need to keep one of them, right?

```suggestion
  cozy-drive = callPackage ../applications/networking/cozy-drive {};

```